src/controllers/controller.bar.js

```diff
@@ -438,7 +446,10 @@ module.exports = DatasetController.extend({
 			start += value;
 		}
 
-		base = vScale.getPixelForValue(start);
+		base = helpers.math._limitValue(vScale.getPixelForValue(start),
```

Comment:
What is this doing?

Reply:
Limits the bar to extend only 10 pixels past chart area. So it does not draw invisible bar with potentially thousands of pixels.

Reply:
Ok. And the reason for 10 is so that you don't see the bar border? Should we use the bar border width here (e.g. in case border is 20 you'd still see it in this case)?

Adding your explanation as a code comment would help as well

Reply:
Added some comments and a TODO about the borderWidth. Its not that trivial, because it can be an object. I don't have the time to do that properly now :)

Reply:
Ok, that sounds fine